modules/shared_utils.py

```diff
@@ -21,6 +22,10 @@ def __repr__(self) -> str:
 
 
 TOKENS: List[Token] = [
+    Token(Owner.BEACON, 100),
```

Comment:
Rather than having this distinguished with a different owner, is it worth having the returned object be more generic? ie it's a `RadioResponse` (hate the name), with `type`, a nulable `owner` and `id`? Feels like the API reflects reality slightly more then

Reply:
For this the previous name of `TargetInfo` would be appropriate.

Reply:
This has now been done in 2782ceb9d0718eab7c129c235aa601c7c5fc1ff2